src/mlpack/tests/cf_test.cpp

```diff
@@ -235,12 +273,17 @@ BOOST_AUTO_TEST_CASE(CFPredictTest)
     }
   }
 
-  // Make data into sparse matrix.
-  arma::sp_mat cleanedData;
-  CF::CleanData(dataset, cleanedData);
+  if (cleanData)
+  {
+    // Make data into sparse matrix.
+    arma::sp_mat cleanedData;
+    CFType<NormalizationType>::CleanData(dataset, cleanedData);
+
+    arma::sp_mat dataset;
+    dataset = cleanedData;
+  }
```

Comment:
I'm not sure if I missed anything because it seems to me that this `if` block doesn't actually do anything.

Reply:
Yeah, I think you can remove it.